timesketch/frontend-ng/src/components/Explore/TimelineChip.vue

```diff
@@ -231,7 +238,6 @@ limitations under the License.
                       <code v-if="datasource.error_message"> {{ datasource.error_message }}</code>
```

Comment:
Somehow this change does mess up the borders for the error messages:
![image](https://github.com/google/timesketch/assets/99879757/bc58ab74-f2ef-40d3-bcc5-50c530bb5f5a)

Reply:
yeah, messy with <pre> tags. I see what we can do here.